### PR TITLE
Update linear-regression-concise.md

### PR DESCRIPTION
**Pull request by emrekaragh**
3.3.9, Alıştırmalar, PyTorch, Alıştırma 1
Sanırım 340. satırdaki şu ifade:
"Eğer `nn.MSELoss(reduction='sum')`ı `nn.MSELoss()` ile değiştirirsek"
bununla değiştirilmelidir:
"Eğer `nn.MSELoss()`ı `nn.MSELoss(reduction='sum')` ile değiştirirsek"

Bu notebook'un 3.3.5. Kayıp Fonksiyonunu Tanımlama bölümünde nn.MSELoss fonskiyonun mevcut kullanımı zaten şu şekilde: 
"loss= nn.MSELoss()"

### CODE REVIEW

**Review by semercim — APPROVED** [review]
@emrekaragh  Thanks for the correction. / Düzeltme için teşekkürler.
